### DISCUSSION TITLE
241219_基本設計議論

### DISCUSSION THREAD

**Discussion by NoRio-kd** [TeamC-by-TNI/Felizo#18] [Announcements]
# 基本設計議論ミーティング 議事録

**日時**: 2023年12月19日 14:00〜14:30  
**場所**: Zoom
**参考資料**: [241221_チャッピー履歴.pdf](https://github.com/user-attachments/files/18216806/241221_.pdf)

## 参加者
- いでっち　@mikiIDE 
- ウッティ    @tkg-geek 
- のり

## 議題
1. 機能について
2. 今後のスケジュール確認
3. その他

### 1. 機能について
- **ハンドルネームとアイコンの生成**:
  - 一旦、ランダムなアルファベットの文字列を表示。
  - 余力があれば偉人などの面白い生成方法を検討。

- **スレッドと自動投稿設定**:
  - スレッドは30分で自動消滅。
  - コメントは10分後に自動削除。
  - スレッド内のトピックは「議題」形式。
  - スタンプ機能や「いいね」は実装可能。Slackのようなオプションがあると良い。
  - 手動削除は一旦許容。
  - 編集機能は非対応（議題が進行すると混乱を招く可能性があるため）。

- **UI/UXの方向性**:
  - ユーザーアイコンは右上には表示しない。
  - `Create` ボタンでスレッド作成。ユーザーアイコンは自動生成または選択の方針は今後決定。
  - 画面上で直接書き込める（モーダルウインドウが望ましい）。
  - スレッド内でユーザーを識別するIDの扱いを検討中（ログインなしの前提で実現可能性を調査）。

- **スタンプ機能**:
  - LINEスタンプの形式ではなく、Slackに近いイメージで進める。

### 2. 今後のスケジュール確認
- 次回MTG: **18:30**
- リリース日程について全員で共有予定。

### 3. その他
- **GitHub運用**:
  - とにかく `develop` に merge していく方針。
  - コードレビューはプルリクエスト時にアナウンスがあった場合のみ実施。

- **役割分担**:
  - メンバーが持てるタスクのイメージを具体化（20〜30個タスクを洗い出し）。
  - うっティが叩き台を作成。

## 決定事項
- 機能の基本方針を確認し、次回MTGで詳細を詰める。
- 次回MTGの日時と内容を確認。

## 次回予定
**日時**: 2024年12月20日 13:00〜14:00  
**場所**: Zoom

## 次回アジェンダ
- 役割分担
- スレッド・コメント機能の仕様確認
- 今後の流れ

**Comment by NoRio-kd** [comment]


**Reply by tkg-geek** [reply]
ありがとうございます！
やるぞやるぞ…！！！
